Modules/Forum/classes/class.ilObjForumGUI.php

```diff
@@ -115,8 +115,7 @@ public function __construct($data, int $id = 0, bool $call_by_reference = true,
                 $this->selected_post_storage->get($thr_pk) ?? 0,
                 $this->is_moderator
             );
-
-            $this->showResetLimitedViewInfo();
+            $this->ctrl->setParameter($this, 'thr_pk', $this->objCurrentTopic->getId());
```

Comment:
Do we really need this line? This seems to be duplicated in https://github.com/ILIAS-eLearning/ILIAS/pull/4672/files#diff-fd85924abce90fbeedabb3e3c4f121dfce9fedd4820943703ccf4a70ab4c31f9R5897

Reply:
Deleting this line will result in error when returning from limited view
with message Could not find first posting by id: 0